src/BaseTemplates/StarterWeb/Views/Shared/Error.cshtml

```diff
@@ -4,3 +4,11 @@
 
 <h1 class="text-danger">Error.</h1>
 <h2 class="text-danger">An error occurred while processing your request.</h2>
+
+<h3>Development Mode</h3>
+<p>
+    Swapping to <strong>Development</strong> environment will display more detailed information about the error that occurred.
+</p>
+<p>
+    Development environment can be enabled by setting the <strong>Hosting:Environment</strong> environment variable to <strong>Development</strong>, and restarting the application.
```

Comment:
Do you know what the name is for RC2? I've been digging thru hosting code and it's a bit of a rabbit maze to work out exactly what env variable controls the name :smile: